### PR TITLE
Restore the useful commits (Idk why they have been reverted)

### PR DESCRIPTION
**Pull request by merinfor2021**
@cclauss and @saloniamatteo , please could you look at this?

### CODE REVIEW

**Comment by saloniamatteo** [comment]
I don't know about this branch, but I deleted the original repository.

I'm not interested in maintaining this project anymore, let alone any other python projects I interacted with and/or I created in the past.

**Comment by merinfor2021** [comment]
@saloniamatteo Sorry for my late reply. No maintenance is requested, of course, _(+ I'm mostly a C guy so not really interested in Python too)_ - but still it would be nice if your project gets restored by this pull request above to its' final state, for everyone else's benefit. Maybe also add a disclaimer " take it as is, won't fix anything "

**Comment by saloniamatteo** [comment]
Hi @merinfor2021,
as I said 6 months ago, I am not interested in this project anymore. No, I won't create another repository.

If you're that desperate, create your own repository. This is a simple script I made **over 4 years ago**, which simply fetches data from PushShift's API, and most of the code simply handles user input. This is trivial to implement. I'm sure by now there are better ways to rewrite this script.

TL;DR Make your own script if you really need it, I don't care. I made this when I was 14, I'm sure you as a full grown man won't have any problem to create a similar project, right?
